visualizations/frontend/filters/v2/TMultiSelector.vue

```diff
@@ -124,7 +124,7 @@
             },
             ids() {
 				const column_name = this.tKeyColumn ? this.tKeyColumn : this.findColumnByTag('ids');
-				return this.getColumn(column_name);
+				return this.getColumn(column_name, 'value');
```

Comment:
I see, rendered has special string formatting applied to it which results in breaking of url, thanks this makes sense.